README.md

```diff
@@ -228,6 +228,29 @@ def main():
   - `resources/templates/baseline` - for templates which the end user **is not** expected to modify.
   - `resources/templates/configurable` - for templates which the end user is expected to modify.
 
+#### Schema validation
+
+Configuration files will be automatically validated against provided schema files whenever `configure apply` is run.
+Validation is done with [jsonschema](https://json-schema.org/) and is only available for `yaml/yml` and `json/jsn` files.
+As an example, the following config file will be valid under the following configuration:
```

Comment:
You should explain it more prior to the example. I.e. be explicit rather than hoping they grok the pattern from the example (naming etc)

I.e. the JSON schema file must be match the name of the file to validate with a suffix of `.schema.json`. It must be placed in the same directory as the file to validate, etc.

Reply:
Does it only apply to files within `resources/templates/configurable/`?
Files anywhere? Files in specific places? etc.